integrations/ahrefs/src/script.raw.js

```diff
@@ -0,0 +1,10 @@
+(function (win, doc) {
+    const trackingID = '<TO_REPLACE>';
+    const js = doc.createElement('script');
+    js.type = 'text/javascript'
+    js.async = true;
+    js.src = 'https://analytics.ahrefs.com/analytics.js';
+    js.dataset['key'] = trackingID;
```

Comment:
I think it should be `data-key` 
```suggestion
    js.dataset['data-key'] = trackingID;
```

Reply:
I believe the [DOMStringMap interface](https://developer.mozilla.org/en-US/docs/Web/API/DOMStringMap) doesn't work that way. According to the [specs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#name_conversion), we are required to:

> Remove the prefix `data-` (including the dash);

Please refer to [these examples](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset#examples) for reference.